docs/site/Controllers.md

```diff
@@ -58,35 +64,54 @@ const spec = {
     '200': {
       description: 'greeting text',
       schema: {type: 'string'},
-    }
-  }
+    },
+  },
 };
 ```
 
-There are several ways to define `Routes` to Controller methods. The first example defines a route to the Controller without any magic.
+There are several ways to define `Routes` to Controller methods. The first
+example defines a route to the Controller without any magic.
 
 ```js
-app.route('get', '/greet', spec, MyController, 'greet');
+// ... in your application constructor
```

Comment:
IDT this has to be in the constructor, does it?

Reply:
With the latest change in `RestApplication`, route registration is now possible in the constructor. Now that I think about it though, there's no guarantee that users will be using or extending from. I know that miroslav's pretty adamant about setting these config stuff in the constructor instead of the `start` function. What do you think?